Model/Rate/Result.php

```diff
@@ -163,37 +151,98 @@ private function addMyParcelRates($parentRate)
             $this->package->setWeightFromQuoteProducts($this->products);
         }
 
-//        foreach ($this->getAllowedMethods() as $alias => $settingPath) {
-//            $settingActive = $this->myParcelHelper->getConfigValue(Data::XML_PATH_BPOST_SETTINGS . $settingPath . 'active');
-//            $active        = $settingActive === '1' || $settingActive === null;
-//            if ($active) {
-//                $method = $this->getShippingMethod($alias, $settingPath, $parentRate);
-//                $this->append($method);
-//            }
-//        }
+        /**
+         * todo:
+         *  - general/enabled must be moved to delivery/active for both carriers
```

Comment:
`- general/enabled must be moved to delivery/active for both carriers` kan weg, is in een andere pull request opgepakt

Reply:
Dan mag je het in die PR weghalen :P

Reply:
Oke dan zal ik dat weg halen